app/models/repository/Merger.scala

```diff
@@ -425,12 +425,21 @@ object Merger {
               balancingCharge = Some(
                 BalancingCharge(
                   balancingChargeYesNo = extractedMaybe
-                    .map(_.raRBalancingChargeYesNo)
+                    .map(_.balancingCharge.isDefined)
```

Comment:
```suggestion
                    .map(_.balancingCharge.exists(_.balancingChargeYesNo == true))
```
Some(false) would be true if we use .isDefined